### ISSUE TITLE
Unexpected behavior in the realizability check

### ISSUE THREAD

**Issue by MrDalekCaan** [NASA-SW-VnV/fret#88]
I've encountered some unexpected behavior with the realizability check. The following requirements are considered realizable:

```
in modeA the system shall always satisfy a
in modeA the system shall always satisfy !a
```
This is the result of realizability check. `modeA` is type of  boolean input and `a` is type of boolean output.
<img width="961" alt="image" src="https://github.com/user-attachments/assets/389bfe4e-2e12-4a6a-9a9c-452b0e58c782">

This result doesn't make sense to me, as these two requirements ask the system to behave in a contradictory manner when in modeA, which should be impossible.

I have a hypothesis regarding this outcome: the realizability check may be verifying whether there exists an instance that can satisfy these requirements. In this case, if the system never enters modeA, both requirements are trivially satisfied, hence passing the check.

My reasoning stems from the following two globally-scoped requirements, which fail the realizability check:

```
the system shall always satisfy a
the system shall always satisfy !a
```
Since these requirements apply globally, no system instance can satisfy them, resulting in the check failing.

Is this behavior intentional or a bug? If it’s intentional, is there a way to enforce a stricter realizability check?

**Comment by andreaskatis** [comment]
Hi @MrDalekCaan,

Thanks for your message. Are you using Kind 2  v2.2.0 by any chance? If yes, you may be observing an issue that we have identified and fixed internally already, caused by changes on the way results are reported after this version. 

Do you observe the same behavior when using Jkind as the engine, or Kind 2 v2.1.1 or older?

Best Regards,

Andreas

**Comment by MrDalekCaan** [comment]
Hi andreaskatis, thank you for you help!

I am using Kind 2 v2.2.0

 Problem solved after switch to Kind 2 v2.1.1

**Comment by andreaskatis** [comment]
Thank you for confirming, @MrDalekCaan.

I will leave this issue open for reference and the fix will be coming very soon with our next release.

Please do not hesitate to reach out if you have further questions.

Best Regards,

Andreas

**Comment by anmavrid** [comment]
The fix has been included in today's release v2.9.1.
